spec/Accessport2LtpTranslator.yaml

```diff
@@ -750,7 +750,7 @@ paths:
                   required: true
                   schema:
                     type: string
-                    example: 'contained-holder(occupying-fru;sequence-id);connector(local-id;sequence-id)'
+                    example: 'contained-holder(occupying-fru;equipment-augment-1-0:holder-pac(sequence-id));connector(local-id;equipment-augment-1-0:connector-pac(sequence-id))'
```

Comment:
the example has been updated correctly, but the response schema and example are incorrect.
The  `"equipment-augment-1-0:holder-pac": { ... }"` and `"equipment-augment-1-0:connector-pac": {...}` parts are missing.
(See example in issue #72).